commit 4f84e7f1f296d58d57cd3bd17345f216e626785e
Author: Kartik Singh <kartik.ynwa@gmail.com>
Date:   Sun Apr 17 22:54:21 2022 +0530

    Enable generating debug symbols

diff --git a/src/Makefile b/src/Makefile
index 30d64be2..751a88e1 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -54,9 +54,9 @@ CCOPT_arm64=
 CCOPT_ppc=
 CCOPT_mips=
 #
-CCDEBUG=
+#CCDEBUG=
 # Uncomment the next line to generate debug information:
-#CCDEBUG= -g
+CCDEBUG= -g
 #
 CCWARN= -Wall
 # Uncomment the next line to enable more warnings:
