commit cf72321ca122b1566b61c9d3f74d6cb973a4c9d8
Author: Kartik Singh <kartik.ynwa@gmail.com>
Date:   Sun Apr 17 22:56:53 2022 +0530

    Fix bcsave for ppc64{,le}

diff --git a/src/jit/bcsave.lua b/src/jit/bcsave.lua
index 6227d136..7aec700c 100644
--- a/src/jit/bcsave.lua
+++ b/src/jit/bcsave.lua
@@ -78,6 +78,8 @@ local map_arch = {
   arm64 =	{ e = "le", b = 64, m = 183, p = 0xaa64, },
   arm64be =	{ e = "be", b = 64, m = 183, },
   ppc =		{ e = "be", b = 32, m = 20, },
+  ppc64 =		{ e = "be", b = 32, m = 20, },
+  ppc64le =		{ e = "be", b = 32, m = 20, },
   mips =	{ e = "be", b = 32, m = 8, f = 0x50001006, },
   mipsel =	{ e = "le", b = 32, m = 8, f = 0x50001006, },
   mips64 =	{ e = "be", b = 64, m = 8, f = 0x80000007, },
